sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -4,7 +4,6 @@
 #nullable enable
 
 using System;
-using System.ClientModel.Primitives;
```

Comment:
Would be good to remove anything else from this file that was added explicitly only for LRO rehydration, unless it's needed by clients that use the shared source files.